apps/src/code-studio/initApp/project.js

```diff
@@ -540,6 +540,12 @@ var projects = module.exports = {
       case 'studio':
         if (appOptions.level.useContractEditor) {
           return 'algebra_game';
+        } else if (appOptions.skin === 'iceage') {
+            return 'iceage';
+        } else if (appOptions.skin === 'infinity') {
+          return 'infinity';
+        } else if (appOptions.skin === 'gumball') {
+            return 'gumball';
```

Comment:
Looks like the `indent` ESLint rule [isn't enabled by default](http://eslint.org/docs/rules/) and [we don't intentionally enable it](https://github.com/code-dot-org/code-dot-org/blob/f0a35e985c3527ec7deb918829874dc7d75c1c8b/.eslintrc.js).